### PR TITLE
Refactor/deploy scripts and tests

### PR DESCRIPTION
**Pull request by gcontarini**
WIP work.
Created some tests for xVault contracts and done small fixes.
Currently, the following methods are tested using optimal flow (meaning: no errors or reverts were tested)
- depositIntoVaults
- withdrawFromVaults
- harvestVaults
- finalizeHarvest

### CODE REVIEW

File: src/tests/utils/SectorTest.sol [hunk 1]
```diff
@@ -10,6 +10,8 @@ contract SectorTest is Test {
 	address owner = address(this);
 
 	address user1 = address(201);
+	// 0x70997970C51812dc3A010C7d01b50e0d17dc79C8
```
Comment (0x0scion) [inline src/tests/utils/SectorTest.sol:13]: what are these?

File: src/vaults/SectorVault.sol [hunk 2]
```diff
@@ -224,12 +224,12 @@ contract SectorVault is SectorBase {
 		/// value here is the fraction of the shares owned by the vault
 		/// since the xVault doesn't know how many shares it holds
 		uint256 xVaultShares = balanceOf(_msg.sender);
-		uint256 shares = (_msg.value * xVaultShares) / 1e18;
+		uint256 shares = (_msg.value * xVaultShares) / 100;
```
Comment (0x0scion) [inline src/vaults/SectorVault.sol:227]: percentages / fractions should always be in the form of `100% = 1e18`

File: src/vaults/SectorVault.sol [hunk 3]
```diff
@@ -224,12 +224,12 @@ contract SectorVault is SectorBase {
...
 		requestRedeem(shares, _msg.sender);
 	}
 
 	function _receiveEmergencyWithdraw(Message calldata _msg) internal {
-		uint256 transferShares = _msg.value.mulWadDown(balanceOf(_msg.sender));
+		uint256 transferShares = (_msg.value * balanceOf(_msg.sender)) / 100;
```
Comment (0x0scion) [inline src/vaults/SectorVault.sol:232]: `/ 1e18`

**Review by 0x0scion — CHANGES_REQUESTED** [review]

File: src/tests/utils/SectorTest.sol (hunk 1 above)
Comment (gcontarini) [reply src/tests/utils/SectorTest.sol:13]: Sorry, totally forgot about it. It's just addresses that I was testing and end up commit it.

File: src/vaults/SectorVault.sol (hunk 2 above)
Comment (gcontarini) [reply src/vaults/SectorVault.sol]: done

File: src/vaults/SectorVault.sol (hunk 3 above)
Comment (gcontarini) [reply src/vaults/SectorVault.sol]: done

**Comment by gcontarini** [comment]
Ready to go!

**Pull request merged**